charts/prometheus-smartctl-exporter/templates/servicemonitor.yaml

```
@@ -5,22 +5,22 @@ metadata:
   name: {{ template "prometheus-smartctl-exporter.fullname" . }}
   labels:
     {{- include "prometheus-smartctl-exporter.labels" . | nindent 4 }}
-{{- if ne (len .Values.serviceMonitor.extraLabels) 0 }}
-{{ toYaml .Values.serviceMonitor.extraLabels | indent 4 }}
-{{- end }}
+    {{- if ne (len .Values.serviceMonitor.extraLabels) 0 }}
+    {{ toYaml .Values.serviceMonitor.extraLabels | indent 4 }}
+    {{- end }}
```

Why not go with `{{ with }}` ?
```suggestion
    {{- with .Values.serviceMonitor.extraLabels }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
```